base_layer/core/src/transactions/transaction_components/memo_field.rs

```diff
@@ -136,7 +136,7 @@ enum InnerMemoField {
     /// Open - the user optionally specifies 'payment_id' ('tx_type' is added by the system).
     Open { payment_id: Vec<u8>, tx_type: TxType },
     /// This payment ID is automatically generated by the system for output UTXOs. The optional user specified
-    /// `PaymentId::Open` payment ID will be assigned to `tx_type` and `payment_id`; the system adds in the sender
+    /// `InnerMemoField::Open` payment ID will be assigned to `tx_type` and `payment_id`; the system adds in the sender
```

Comment:
```suggestion
    /// `MemoField::Open` payment ID will be assigned to `tx_type` and `payment_id`; the system adds in the sender
```
InnerMemoField is for the code only
`///` is for doc strings, mentioning of InnerMemoField  in docs is not correct, this is for the inner working of the class. 
Mentioning of InnerMemoField  in `//` which is comments is fine, but not doc strings.

Reply:
The entire `enum InnerMemoField {` has doc strings in it; I will change all of them.